mapie/multi_label_classification.py

```diff
@@ -370,7 +414,42 @@ def _check_bound(self, bound: Optional[str]):
             warnings.warn(
                 "WARNING: you are using crc method, hence "
                 + "even if the bound is not ``None``, it won't be"
-                + "taken into account"
+                + "taken into account. "
```

Comment:
STYLE: one whitespace should be removed.

Reply:
Done